test/test_mypy.py

```diff
@@ -277,6 +295,10 @@ def foo(self):
 
 
 class TestDocumentType(unittest.TestCase):
+    def setUp(self) -> None:
+        if TypedDict is None:
+            raise unittest.SkipTest("Python 3.8+ is required to use TypedDict.")
+
```

Comment:
This skip is overeager. It will skip the below tests that don't even use TypedDict. Can we remove this?